packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -108,27 +108,8 @@ import { EntityTodoContent } from '@backstage/plugin-todo';
 import { Button, Grid } from '@material-ui/core';
 import BadgeIcon from '@material-ui/icons/CallToAction';
 
-import {
-  EntityGithubInsightsContent,
-  EntityGithubInsightsLanguagesCard,
-  EntityGithubInsightsReadmeCard,
-  EntityGithubInsightsReleasesCard,
-  isGithubInsightsAvailable,
-} from '@roadiehq/backstage-plugin-github-insights';
-import {
-  EntityGithubPullRequestsContent,
-  EntityGithubPullRequestsOverviewCard,
-  isGithubPullRequestsAvailable,
-} from '@roadiehq/backstage-plugin-github-pull-requests';
-import {
-  EntityTravisCIContent,
-  EntityTravisCIOverviewCard,
-  isTravisciAvailable,
-} from '@roadiehq/backstage-plugin-travis-ci';
-import {
-  EntityBuildkiteContent,
-  isBuildkiteAvailable,
-} from '@roadiehq/backstage-plugin-buildkite';
```

Comment:
all of this should be reverted right?

Reply:
Yes, I made it a separate commit so that it's easy to revert once fixed in upstream plugins.

Reply:
@dtuite 👀